README.md

```diff
@@ -112,6 +112,13 @@ sudo systemctl restart snap.microk8s.daemon-docker.service
 
 To troubleshoot a non-functional microk8s deployment you would first check that all of the above services are up and running. You do that with `sudo systemctl status snap.microk8s.<daemon>`. Having spotted a stopped service you can look at its logs with `journalctl -u snap.microk8s.<daemon>.service`. We will probably ask you for those logs as soon as you open an [issue](https://github.com/juju-solutions/microk8s/issues).
 
+### Firewall configuration and network security
```

Comment:
I would do this instead:

### Troubleshooting
#### My dns and dashboard pods are CrashLooping.
The [Kubenet](https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/) network plugin used by microk8s creates a `cbr0` interface when the first pod is created. If you have `ufw` enabled, you'll need to allow traffic on this interface:
```
sudo ufw allow in on cbr0 && sudo ufw allow out on cbr0
```

We can add more troubleshooting tips here as needed.